cltk/prosody/old_norse/verse.py

```diff
@@ -589,3 +591,57 @@ def find_alliteration(self):
         :return:
         """
         return Metre.find_alliteration(self)
+
+
+class PoetryTools:
+    def __init__(self, basic=True):
+        if basic:
+            self.syllabifier = Syllabifier(language="old_norse_ipa")
+            self.tr = Transcriber(DIPHTHONGS_IPA, DIPHTHONGS_IPA_class, IPA_class, old_norse_rules)
+            self.tagger = POSTag('old_norse')
+        else:
```

Comment:
In this PR, will you add code for `basic=False`? If not, then I would prefer that you only keep the `basic` code and remove the condition (and parameter) entirely.

Reply:
I was not sure about it. I will remove the parameter.

Reply:
Done.